test/otherTools.cpp

```diff
@@ -252,11 +253,14 @@ TEST(networkTools, getNetworkInterfaces)
   }
 }
 
-TEST(networkTools, getBestPublicIp)
+TEST(networkTools, getBestPublicIps)
 {
+  using kiwix::getBestPublicIps;
   using kiwix::getBestPublicIp;
+  using kiwix::IpMode;
 
-  std::cout << "getBestPublicIp(true)  " << getBestPublicIp(true)  << std::endl;
-  std::cout << "getBestPublicIp(false) " << getBestPublicIp(false) << std::endl;
-  std::cout << "getBestPublicIp()      " << getBestPublicIp()      << std::endl;
+  std::cout << "getBestPublicIps(IpMode::ipv4) : [" << getBestPublicIps(IpMode::ipv4).addr << ", " << getBestPublicIps(IpMode::ipv4).addr6 << "]" << std::endl;
+  std::cout << "getBestPublicIps(IpMode::ipv6) : [" << getBestPublicIps(IpMode::ipv6).addr << ", " << getBestPublicIps(IpMode::ipv6).addr6 << "]" << std::endl;
+  std::cout << "getBestPublicIps(IpMode::all)  : [" << getBestPublicIps(IpMode::all).addr << ", " << getBestPublicIps(IpMode::all).addr6 << "]" << std::endl;
```

Comment:
Please overload `operator<<(std::ostream&, kiwix::IpAddress&)` above this test and make this code more readable.